lib/matplotlib/font_manager.py

```diff
@@ -1461,16 +1569,51 @@ def _get_font(filename, hinting_factor, *, _kerning_factor, thread_id):
     os.register_at_fork(after_in_child=_get_font.cache_clear)
 
 
-def get_font(filename, hinting_factor=None):
+@lru_cache(64)
+def _cached_realpath(path):
     # Resolving the path avoids embedding the font twice in pdf/ps output if a
     # single font is selected using two different relative paths.
-    filename = _cached_realpath(filename)
+    return os.path.realpath(path)
+
+
+@_api.rename_parameter('3.6', "filepath", "font_filepaths")
+def get_font(font_filepaths, hinting_factor=None):
+    """
+    Get an `ft2font.FT2Font` object given a list of file paths
+
+    Parameters
+    ----------
+    font_filepaths : Iterable[str|Path|byhen] | str | Path | bytes
```

Comment:
```suggestion
    font_filepaths : Iterable[str|Path|bytes] | str | Path | bytes
```
but do we normally do the type hint `|`? Isn't it normally a comma?